src/pb_preferred_memory.h

```diff
@@ -30,10 +30,10 @@ namespace triton { namespace backend { namespace python {
 
 class PreferredMemory {
  public:
-  enum MemoryType { GPU, CPU, DEFAULT };
+  enum MemoryType { kGPU, kCPU, kDefault };
```

Comment:
What's the significance of all the `CONSTANT` -> `kConstant` changes? Is it a windows requirement? Or style change?

Reply:
`ERROR` is defined as a macro when we include `windows.h`, so there is a naming collision. This would also be true for names such as `IN` or `OUT`. Probably for this reason, Google's style sheet says enum fields should be named like constants and not macros, so I went ahead changed how they are declared universally in the pybe.